ieeeuoft/hackathon-template
# IEEE-221: add phone number to profile model and update all subsequent serializers

## Overview

- added phone_number field to profile model with validation
- updated serializers
- updated unit tests


## Unit Tests Created

- none


## Steps to QA

- apply migrations (you will probably have to provide a default phone number when migrating the profiles that already exist in your local db, since the field is required and has no default)
- check the django admin page and test out some of the relevant event/profile apis on swagger to see that the phone number is now there
- `event.test_api.CreateProfileViewTestCase.test_user_can_have_profile` is still failing and i can't figure out why
